ionic_user_interface/src/components/wall-image-viewer/Canvas.vue

```diff
@@ -22,15 +22,14 @@
         <ion-label>MarkDone</ion-label>
       </ion-segment-button>
     </ion-segment>
-    <ion-button
-      v-if="+selectedMode === SelectMode.MARKDONE"
-      @click="handleExportClick"
-      class="solid-button"
-      fill="solid"
-      color="secondary"
-    >
-      Export
-    </ion-button>
+    <div v-if="+selectedMode === SelectMode.MARKDONE">
+      <ion-button @click="handleExportClick" class="solid-button" fill="solid" color="secondary">
+        Export
+      </ion-button>
+      <ion-button @click="handleUploadClick" class="solid-button" fill="solid" color="primary">
+        Upload
+      </ion-button>
+    </div>
```

Comment:
I think `EXPORT` can be `fill="clear"` so that the user will know the more natural thing will be to upload.

But instead of `UPLOAD`, should we change to `POST`, more essence of sharing?